devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -205,17 +201,18 @@ def configure() {
 	// enrolls with default periodic reporting until newer 5 min interval is confirmed
 	sendEvent(name: "checkInterval", value: 2 * 10 * 60 + 1 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
 
-	// OnOff minReportTime 0 seconds, maxReportTime 5 min. Reporting interval if no activity
-	refresh()
+	zigbee.setColorTemperature(5000) +
```

Comment:
@KKlimczukS I thought of one more issue last night. `configure` can be run after initial inboarding so it's possible the color temp is currently at the desired value and this changes it to a different value. What do you think about only setting the color temp to 5000 if the attribute doesn't have a value?

Reply:
@tpmanley The issue was reported because color temperature was set to 6.536 K to right after pairing. If we added the condition You mentioned it would work for general cases where the set values are in range but i'm afraid it wouldn't fix the reported problem.

Reply:
@KKlimczukS my suggestion is to set the value only if there isn't a current value meaning it will only be run when the device is initially onboarded so I think it will fix the reported problem.

Reply:
@tpmanley ok, fixed, please let me know if that looks better now.